protocol/x/subaccounts/keeper/subaccount_helper.go

```diff
@@ -106,7 +106,6 @@ func UpdatePerpetualPositions(
 	perpIdToFundingIndex map[uint32]dtypes.SerializableInt,
 ) (
 	success bool,
```

Comment:
Hmm actually on re-reading this function, this always returns `success = true`. I'm going to remove this variable as well.

Reply:
Done!